frontend/catalyst/api_extensions/quantum_operators.py

```diff
@@ -147,17 +153,14 @@ def circuit():
             f"Measure is only supported on 1 qubit, got array of shape {wires[0].shape}."
         )
 
-    # Copy, so wires remain unmodified
-    in_classical_tracers = wires.copy()
-
     if postselect is not None and postselect not in [0, 1]:
         raise TypeError(f"postselect must be '0' or '1', got {postselect}")
 
     m = new_inner_tracer(ctx.trace, get_aval(True))
     MidCircuitMeasure(
-        in_classical_tracers=in_classical_tracers,
-        out_classical_tracers=[m],
-        regions=[],
+        # Copy, so wires remain unmodified
+        wires=wires.copy(),
```

Comment:
Not particularly. Now that I also think about it, we don't need it since the wires get cast inside a `Wires` object now so we wouldn't be able to mutate them anyway.